cpp/locality/NearestNeighbors.cc

```diff
@@ -51,18 +51,18 @@ void NearestNeighbors::compute(const box::Box& box, const vec3<float>* ref_pos,
                                const vec3<float>* pos, unsigned int num_points, bool exclude_ii)
```

Comment:
This is another class that we plan to remove. We should probably do some benchmarking to compare this class against the query methods for AABBQuery and LinkCell first, but then I think we're in good shape to get rid of this as well.